City-of-Helsinki/drupal-helfi-platform-config
# UHF-1872 url list units

How to test:
- Setup Sote -site using [THIS SOTE BRANCH](https://github.com/City-of-Helsinki/drupal-helfi-sote/tree/UHF-1872_linkit_list_tpr_entities)
- Install the site: `make new`
- `composer require drupal/helfi_platform_config:dev-UHF-1872_url_list_units`
- `composer require drupal/hdbt_admin:dev-UHF-1872_linkit_list_tpr_entities`
- Run make drush-updb
- Run make drush-cex
- Run inside container `MIGRATE_LIMIT=10 drush mim tpr_unit`
- Run inside container `MIGRATE_LIMIT=5 drush mim tpr_service`
- run drush cr

What you should see:
Go to /admin/structure/paragraphs_type/banner/form-display and open the link field settings from the cog icon
- Linkit profile dropdown value should be "TPR/content"
Go to create new basic page and add a banner
- If you start typing to url field, the field should get autocomple-values from tpr_units & tpr_services (and content if you have any)
